pkg/internal/request/span.go

```diff
@@ -28,13 +31,59 @@ const (
 	EventTypeKafkaServer
 )
 
+func (t EventType) String() string {
```

Comment:
don't we have a similar function in `debug.go` (`spanType`)? or this one is will replace the other?

Reply:
We actually do, you are right! They return different strings though, the `debug.go` one being `ALL_CAPS` truncated versions, this one being more "json friendly". 
It may be a good idea to merge them indeed - do you have a preference on which version to use? (I prefer the "new" one but I am worried using it may break legacy code, so perhaps not? )

Reply:
I don't have any preference and I think merging it's fine. I don't thing it will break anything, it's just a printed string.

Reply:
Cool beans, I'll merge it!